src/test/rustdoc-gui/mobile.goml

```diff
@@ -19,7 +18,6 @@ assert-property: (".mobile-topbar h2.location", {"offsetHeight": 36})
 assert-css: (".content .out-of-band .since::before", { "content": "\"Since \"" })
 
 size: (1000, 1000)
-wait-for: 100 // wait a bit for the resize to be fully taken into account.
```

Comment:
I would prefer to avoid bundling this change in -- it seems unrelated to this PR and IIRC was a mitigation for spurious failures?

Reply:
It's to confirm my changes are working. I'll remove it once the CI pass.